setup/MSBuild.vsmanproj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <Import Project="$(MicroBuildDir)MicroBuild.Core.props" />
+
+  <PropertyGroup>
+    <!-- Define properties that drive the manifest creation here. -->
+    <FinalizeManifest>true</FinalizeManifest>
+    <FinalizeSkipLayout>true</FinalizeSkipLayout>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutDir>$(BaseOutputPath)setup\</OutDir>
+    <OutputPath>$(OutputPath)setup\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="MSBuild.swixproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <MergeManifest Include="$(BaseOutputPath)setup\*.json" />
```

Comment:
Good catch! Eventually these two swix and vsman projects can be merged and I'll make sure that's taken into account. For now they're just called separately from a build definition and not meant to be consumed outside of that.